sosia/establishing/database.py

```diff
@@ -1,11 +1,13 @@
 """This module provides functions for connecting to and creating a SQLite database."""
 
+from pathlib import Path
```

Comment:
One love down please

Reply:
Just one extra line before `from pathlib import Path`?

Reply:
Nope. The order is: within the block of base packages, first the full imports alphabetically, then the partial imports, also alphabetically.

Reply:
Got it